code/modules/urist/items/clothstuff.dm

```diff
@@ -94,3 +94,46 @@
 				else
 					to_chat(user, "<span class='warning'>\The [src] is used up, but there are more wounds to treat on \the [affecting.name].</span>")
 			use(used)
+
+/obj/item/loom
```

Comment:
for that matter, is there a particular reason why this is an item and not a structure? It seems like it makes a lot more sense as a structure, particularly with the sprite as it is.

Reply:
because i dont know how to make cotton used on the loom turn into cloth

Reply:
also bc the loom is just not a good sprite so at least as a table loom item instead of a floor loom, it can be moved around and be out of the way